### ISSUE TITLE
More Lesson Viewer Differentiations?

### ISSUE THREAD

**Issue by sgoshorn** [CCALI/CALI-Author-Viewer-5#52]
Should we be differentiating the lesson viewer per source of lesson? There's the main lesson,  AutoPublish logo and there will be a QuizWright logo shortly. But should also have one for LessonLink? Other logos/colors for other purposes, like a Demo version scheme?

**Comment by johnmayer** [comment]
Yes.  At the very least, so that we can diagnose problems, so something in the interface - even subtle.

**Comment by emasters** [comment]
I think it needs to be _real_ obvious. We can already track problems around this. This is a UI/UX issue, so students and faculty know where they are and that they are doing the thing they should be doing.
